cmd/node/config/systemSmartContractsConfig.toml

```diff
@@ -20,11 +20,17 @@
     EnabledEpoch = 2
 
 [GovernanceSystemSCConfig]
-	ProposalCost = "5000000000000000000" #5 eGLD
-	MinQuorum    = "158000000000000000000" # sqrt(10*NodePrice)
-	MinPassThreshold = "80000000000000000000"
-	MinVetoThreshold = "30000000000000000000"
-    EnabledEpoch = 2
+    NumNodes           = 500
+    ProposalCost       = "5000000000000000000" #5 eGLD
+    ProposalCostV2     = "5000000000000000000" #5 eGLD
+	MinQuorum          = 400
+	MinQuorumV2        = "500000000000" # VotePower of 10*sqrt(NodePrice)
+	MinPassThreshold   = 300
+	MinPassThresholdV2 = "251000000000"
```

Comment:
This is confusing, Why not we should have GovernanceSystemSCConfigV2 group instead?

Reply:
It was a discussion with @sasurobert about this, regarding how are the rest of the V2 configs handled. So I went the same way.

Reply:
Ok, but we should have different structures for different versions of the config. Otherwise we will end up having a config structure of hundreds of lines.

Reply:
done